src/test/scala/modbat/mbt/ModbatTestHarness.scala

```diff
@@ -35,8 +61,7 @@ object ModbatTestHarness {
   }
 
   def compare_iterators(it1:Iterator[String], it2:Iterator[String]) : Boolean = {
-    var current_iterator="There is no element in the Iterator."
-    var current_iterator2="There is no element in the Iterator."
+    var current_iterator, current_iterator2="There is no element in the Iterator."
```

Comment:
Use only *one* "current" (past) iterator (for the case when the lines match); change the text to "End of file."